api_v3/lib/types/entry/filters/base/KalturaBaseEntryBaseFilter.php

```diff
@@ -76,7 +76,9 @@
 		"rootEntryIdIn" => "_in_root_entry_id",
 		"parentEntryIdEqual" => "_eq_parent_entry_id",
 		"entitledUsersEditMatchAnd" => "_matchand_entitled_users_edit",
+		"entitledUsersEditMatchOr" => "_matchor_entitled_users_edit",
```

Comment:
Please add it also in KalturaBaseEntryFilter and handle the translation from puser-id to kuser-id in fixFilterUserId.